hash/apple2_flop_orig.xml

```diff
@@ -22537,7 +23056,8 @@ license:CC0-1.0
 		<description>Guardian (Continental Software)</description>
 		<year>1982</year>
 		<publisher>Continental Software</publisher>
-		<info name="programmer" value="Tom and Jerry" />
+		<info name="programmer" value="Tom" />
+		<info name="programmer" value="Jerry" />
```

Comment:
I think “Tom and Jerry” is the name of a team.  (Also, Bob Flanagan and Scott Miller supposedly worked on this but don’t appear in the credits.)